README.md

```diff
@@ -43,17 +43,28 @@ rails db:setup
 
 #### Developing with the Android app
 
-To run [simple-android](https://github.com/simpledotorg/simple-android/) app
-with the server running locally, you can use [ngrok](https://ngrok.com).
+To run [simple-android](https://github.com/simpledotorg/simple-android/) app with the server running locally, you can
+use [ngrok](https://ngrok.com).
 
 ```bash
 brew cask install ngrok
 rails server
 ngrok http 3000
 ```
 
-The output of the ngrok command will have an url that can be used to access
-local-server. This url should be set as `qaApiEndpoint` in gradle.properties.
+The output of the ngrok command is HTTP and HTTPS URLs that can be used to access your local server. Configure the
```

Comment:
Might be good to mention that the HTTP URL should be ignored, as it won't work due to security policies ( I figured this out the hard way 😏 ).